tests/js/server/replication2/replication2-soft-write-concern-replicated-log-cluster.js

```diff
@@ -28,48 +28,87 @@ const {assertEqual} = jsunity.jsUnity.assertions;
     writeConcern: 2,
     softWriteConcern: 3,
-    waitForSync: false,
+    waitForSync: waitForSync,
   };
 
+  const replicationFactor = 3;
+
   const {setUpAll, tearDownAll, stopServerWait, continueServerWait, setUp, tearDown} =
     helper.testHelperFunctions(database, {replicationVersion: "2"});
 
   const createReplicatedLogAndWaitForLeader = function (database) {
-    return helper.createReplicatedLog(database, targetConfig);
+    return helper.createReplicatedLog(database, targetConfig, replicationFactor);
   };
 
   return {
     setUpAll, tearDownAll,
     setUp, tearDown,
 
+    testLogStartWithMissingFollower: function () {
+      // TODO(CINFRA-793) this test does not work for waitForSync false
+      let servers = _.sampleSize(helper.dbservers, replicationFactor);
+      stopServerWait(servers[0]);
+
+      // We should be able to create the log and use it even when one of the servers is unavailable.
+      const {logId} = helper.createReplicatedLogInTarget(database, targetConfig, replicationFactor, servers);
+      let log = db._replicatedLog(logId);
+      let quorum = log.insert({foo: "bar"});
+      assertEqual(quorum.result.quorum.quorum.length, 2);
+    },
+
     testParticipantFailedOnInsert: function () {
       const {logId, servers, leader, term, followers} = createReplicatedLogAndWaitForLeader(database);
-      // This test may not be stable. When a log is created, the followers
-      // haven't yet reported to Current. The Supervision may immediately lower
-      // the effectiveWriteConcern, because it assumes the participants are
-      // unavailable.
-      // The following waits for the followers to report to Current. However,
-      // there might still be a race with the Supervision.
-      // If the test fails because the quorum size is only 2, this is probably
-      // the reason: Please report such a failure to team CINFRA, so we can try
-      // again to stabilize the test.
+      // When a log is created, the followers haven't yet reported to Current.
+      // The Supervision may immediately lower the effectiveWriteConcern,
+      // because it assumes the participants are unavailable.
+      // The following waits for the followers to report to Current.
       waitFor(lp.replicatedLogIsReady(database, logId, term, servers, leader));
 
+      // Due to the writeConcern of 2, the leader may have established leadership when
+      // only one of the followers was ready. While the committedParticipantsConfig's
+      // effectiveWriteConcern becomes 2, the leadership has been established without
+      // a 3rd participant. It is therefore possible that the next insert will be
+      // committed with a quorum of 2.
+      waitFor(() => {
+        let {plan} = helper.readReplicatedLogAgency(database, logId);
+        if (plan.participantsConfig.config.effectiveWriteConcern !== replicationFactor) {
+          return Error(`effectiveWriteConcern not reached, found ${JSON.stringify(plan)}`);
```

Comment:
```suggestion
          return Error(`effectiveWriteConcern not reached, found ${JSON.stringify(plan)}, expected ${replicationFactor}`);
```

Reply:
Applied in https://github.com/arangodb/arangodb/pull/19375